pupil-spa/src/app/services/app-usage/app-usage.service.ts

```diff
@@ -1,12 +1,22 @@
 import { Injectable } from '@angular/core';
+import { StorageService } from '../storage/storage.service';
+import { DeviceService } from '../device/device.service';
+
 
 @Injectable()
 export class AppUsageService {
 
-  private appUsageCounter;
+  private appUsageCounter: number;
+
+  constructor(private storageService: StorageService) {
+    const deviceData = storageService.getItem(DeviceService.storageKey);
```

Comment:
we decided earlier this week not to use localStorage, as it seemed overkill.  are you doing this because you are unable to persist the counter across page reloads / re-logins?